tests/installation/add_maintenance_repos.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Add new add-ons using maintenance test repo URLs.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi 'get_var';
+
+sub run {
+    my @repos = split(/,/, get_var('MAINT_TEST_REPO'));
+
+    $testapi::distri->get_add_on_product()->confirm_like_additional_add_on();
+
+    my $maintrepo = shift @repos;
+    $testapi::distri->get_add_on_product()->accept_current_media_type_selection();
+    $testapi::distri->get_repository_url()->add_repo({url => $maintrepo});
```

Comment:
we still need to add the name of the repo.
see suggestion: https://progress.opensuse.org/issues/113731
we did for AutoYaST because it might help to investigate and it is cheap to type it with libyui-rest-api.

Reply:
Sure, adding a name for the repo is super easy... but where would the repo names be coming from? I mean, `MAINT_TEST_REPO` does not include repo names, only urls. Can you point me to the way it's done in autoyast?

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15392#discussion_r954548934

Reply:
now first repo doesn't have name. feel free to extract the function on top.

Reply:
done!